cli/src/commands/workspaces/create.ts

```diff
@@ -7,9 +8,24 @@ export default class WorkspacesCreate extends BaseCommand {
 
   static flags = {};
 
-  static args = [];
+  static args = [{ name: 'workspace', description: 'The new workspace name', required: true }];
 
-  async run(): Promise<void> {
-    this.error('To be done');
+  static enableJsonFlag = true;
+
+  async run(): Promise<void | unknown> {
+    const { args } = await this.parse(WorkspacesCreate);
+    const { workspace } = args;
+
+    if (!workspace) {
+      return this.error('Please, specify a workspace name');
+    }
+
+    const xata = await this.getXataClient();
+
+    const result = await xata.workspaces.createWorkspace({ name: workspace, slug: slugify(workspace) });
```

Comment:
Do we want to allow the user to send through flags more parameters like slug? Even if we default to the slugify

Reply:
Good suggestion!

Another option is that the backend makes the `slug` field optional and is calculated in the server side. But anyway, maybe it's a good idea to allow users to specify it. I'll add to the _improvements_ issue.